src/com/pentalog/pentastagiu/homework2/registration/Board.java

```diff
@@ -0,0 +1,22 @@
+package com.pentalog.pentastagiu.homework2.registration;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Board {
+    public static List<String> messageList = new ArrayList<>();
```

Comment:
ai adaugat lista de mesaje aici, dar nu te-ai folosit nicaieri de ea, ramane mereu goala; este o idee buna dpdv tehnic sa pastezi aici o lista de mesaje; astfel daca ai avea o logica mai complexa ai putea sa aranjezi mesajele in functie de ora la care au fost postate sau de autor
+ nu e nevoie sa adaugi si list, poti folosi pt o colectie direct pluralul: messages in cazul asta